build/build.sh

```diff
@@ -232,6 +239,10 @@ edge_dir=${EDGE_DIR:-Edge}
 # Disable with '-disable-werror' option.
 cmake_flags+=" -DXRT_ENABLE_WERROR=$werror"
 
+# set CMAKE_INSTALL_PREFIX
+export XRT_INSTALL_PREFIX=$xrt_install_prefix
```

Comment:
Why export?  This script is not sourced, it is used only for invoking the build.  Isn't it sufficient to set CMAKE_INSTALL_PREFIX to xrt_install_prefix?  Do we need the XRT_INSTALL_PREFIX variable at all?

Reply:
I thought it would be simpler to have a separate variable XRT_INSTALL_PREFIX for XRT install location. I will modify it to use CMAKE_INSTALL_PREFIX only.